vmware_aria_operations_integration_sdk/adapter_config.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import List
+
+
+class Question:
+    def __init__(self, key: str, prompt: Callable, *args: Any, **kwargs: Any) -> None:
+        self.key = key
+        self.prompt = prompt
+        self.prompt_args = args
+        self.prompt_kwargs = kwargs
+
+    def ask(self) -> Any:
+        return self.prompt(*self.prompt_args, **self.prompt_kwargs)
+
+
+class AdapterConfig:
+    def __init__(self, language: str, questions: List[Question] = []) -> None:
+        self.language = language
+        self.values: Dict[str, Any] = dict()
+        self.questions = questions
+
+    def prompt_config_values(self) -> None:
+        for question in self.questions:
+            self.values[question.key] = question.ask()
```

Comment:
This is nice. I think as we refactor the init code, anywhere where we're branching on AdapterConfig::language is a good candidate to move the method into this class 👍